packages/zowe-explorer/src/utils/ReleaseNotes.ts

```diff
@@ -0,0 +1,182 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { WebView } from "@zowe/zowe-explorer-api";
+import { ExtensionContext, l10n, ViewColumn } from "vscode";
+import { ZoweLogger } from "../tools/ZoweLogger";
+import * as fs from "fs/promises";
+import { SettingsConfig } from "../configuration/SettingsConfig";
+import { Constants } from "../configuration/Constants";
+import { ZoweLocalStorage } from "../tools/ZoweLocalStorage";
+import { Definitions } from "../configuration/Definitions";
+
+export class ReleaseNotes extends WebView {
+    private version: string;
+    private readonly extensionVersion: string;
+    public static instance: ReleaseNotes | undefined;
+
+    public static getExtensionVersion(context: ExtensionContext): string {
+        // Get extension version (major.minor)
+        const extensionVersion: string = context.extension.packageJSON.version;
+        const versionRegex = /(\d+\.\d+)/;
+        const majorMinorVersion = extensionVersion.match(versionRegex);
+        return majorMinorVersion ? majorMinorVersion[0] : extensionVersion;
+    }
+
+    public static shoulddisplayReleaseNotes(context: ExtensionContext): { version: string; displayReleaseNotes: boolean } {
```

Comment:
(nitpick) This is likely a typo but we should use `camelCase` for function names:

```suggestion
    public static shouldDisplayReleaseNotes(context: ExtensionContext): { version: string; displayReleaseNotes: boolean } {
```